src/AppKit/Enums.cs

```diff
@@ -1929,24 +2950,44 @@ public enum NSWorkspaceLaunchOptions : ulong {
 		Default = Async | AllowingClassicStartup
 	}
 
+#if NET
+	[UnsupportedOSPlatform ("maccatalyst")]
+#else
 	[NoMacCatalyst]
+#endif
 	[Flags]
 	[Native]
 	public enum NSWorkspaceIconCreationOptions : ulong {
 		NSExcludeQuickDrawElements   = 1 << 1,
 		NSExclude10_4Elements       = 1 << 2
 	}
 
+#if NET
+	[UnsupportedOSPlatform ("maccatalyst")]
+#else
 	[NoMacCatalyst]
+#endif
 	[Native]
 	public enum NSPathStyle : long {
 		Standard,
+#if NET
+		[UnsupportedOSPlatform ("macos10.7")]
```

Comment:
I think min mac version is 10.14, would that affect this?

Reply:
Or maybe is this okay and we just don't want to say that it is available before 10.14?

Reply:
My plan right now is to convert all attributes as is, to reduce the surface for mistake. In the future we'll have to extend the attribute stripper to strip out "old" attributes as every year we'll be dropping a version".

Reply:
No need, we already did that in several places: https://github.com/xamarin/xamarin-macios/blob/main/src/CoreText/CTFontManager.cs#L81 and that was you @tj-devel709  ;) 

5b896c7789 (TJ Lambert         2021-09-01 11:30:34 -0500 101)